### PR TITLE
[enterprise-4.13] edit toolbox deprecation notice

### PR DESCRIPTION
**Pull request by jldohmann**
Version(s): 4.13

Issue: n/a

Link to docs preview: http://file.rdu.redhat.com/jdohmann/minor-RN-edit/release_notes/ocp-4-13-release-notes.html#ocp-4-13-toolbox-deprecation

QE review:
- [x] QE has approved this change.

Additional information: fixes the toolbox deprecation notice in #57453

### CODE REVIEW

**Comment by jldohmann** [comment]
@travier could you PTAL and lmk what you think? thanks!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -574,7 +574,7 @@ For more information on converting existing ICSP YAML files to IDMS YAML files,
 
 [id="ocp-4-13-toolbox-deprecation"]
 ==== Toolbox is deprecated in {op-system}
-The toolbox library is deprecated and support will be removed from a future {product-title} release.
+The toolbox script is deprecated and support will be removed from a future {product-title} release and replaced with the sos package.
```
Comment (travier) [inline release_notes/ocp-4-13-release-notes.adoc:577]: Well, we're not replacing it with the sos package. We either won't replace it at all or replace it with the containers/toolbox one but it's not decided yet.

```suggestion
The toolbox script is deprecated and support will be removed from a future {product-title} release.
```

**Comment by nalhadef** [comment]
label peer-review-in-progress
/remove-label peer-review-needed

**Comment by mike-nguyen** [comment]
LGTM

**Pull request merged**